.github/workflows/tests.yml

```diff
@@ -26,8 +26,7 @@ jobs:
       - name: Install dependencies
         run: |
           apt-get update && apt-get install -y shellcheck
-          pip install -r requirements-build.txt
-          pip install -r requirements.txt
+          pip install -r requirements.txt -r requirements-build.txt
```

Comment:
Install both requirements files in the same command line.
This captures that requirements interactions between the 2 a little better.